src/app/ui-kit/Button/index.tsx

```diff
@@ -54,7 +55,8 @@ const buttonVariants = css<ButtonProps>`
           border: 1px solid #000;
           transition: all 0.3s ease;
           &:hover {
-            background-color: #333;
```

Comment:
Hey @tarun. I guess you meant to assign opposite variables

background-color: ${backgroundcolor};
color: ${color};

Reply:
Rest LGTM.